.vscode/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $# -ne 1 ]]; then
+  echo "Invalid test target arguments.
+
+Please invoke test from VSCode via the workbench.action.tasks.test command."
+  exit 1
+fi
+
+if [[ "$1" != *.dart ]]; then
+  echo "Only .dart files can be tested by task in this repo."
+  exit 1
+fi
+
+echo "Testing $1..."
+
+if [[ "$1" == packages/flutter/test/* ]]; then
+  cd packages/flutter
+  ../../bin/flutter test ../../$1
```

Comment:
Are these paths correct? Seems strange that one starts with `bin` but the other `../../bin` when they both refer to the same `bin` folder? (I'm on Windows so can't test atm...)

I could try making a Windows version of the script, but, I think when we do https://github.com/Dart-Code/Dart-Code/issues/499 this might all get much simpler (needs a little investigation) and won't require maintaining a shell script like that.

Reply:
the flutter execution needs to be run in the directory with the pubspec.yaml. 

the dart executable can just be run from the flutter repo root. 

Mostly just a convenience script for us for now. Dart-Code/Dart-Code#499 is definitely a superior solution.

Reply:
Oh wow, I missed the `cd` on the line before... Monday morning!

LGTM then! If I end up working in here from Windows before Dart-Code/Dart-Code#499 I might create a Windows `.bat` equiv, but I hope the tasks isn't too far off!